### ISSUE TITLE
Plone 6.0.0a2 tinymce pattern, in '/en' the visualblocks plugin cannot load the css because of a wrong url

### ISSUE THREAD

**Issue by yurj** [plone/mockup#1102]
I've found a bug with multilingual (or '/en' folder) and tinymce. If you enable the visual blocks (view -> Show Blocks), in an english site the tinymce-visualblocks css url miss the ++plone++static part :`http://<mysite>/++plone++static/components/tinymce-builded/js/tinymce/plugins/visualblocks/css/visualblocks.css` vs `http://<mysite>/en//plugins/visualblocks/css/visualblocks.css` (the // misses ++plone++static plus all the path to the plugin).

The bug happen if the folder is called, for example, 'en' and language negotiation (folder and browser) is enabled in the control panel, and the language 'en' is added to available languages. If I call the folder with a different name, or with a non installed language, or if I disable the language negotiation, then the css is loaded correctly, the css url in the head is `http://mysite/++plone++static/components/tinymce-builded/js/tinymce/plugins/visualblocks/css/visualblocks.css`

**Comment by yurj** [comment]
https://github.com/plone/plone.base/pull/18 maybe has anything to do with it?

**Comment by Rudd-O** [comment]
I would like to help you fix this but I need some guidance.  What's the visualblocks plugin and how do I enable it on my TinyMCE?

**Comment by Rudd-O** [comment]
Actually, the `tinymce-builded` folder is gone from the buildout (as of Plone 6 beta).  Not sure how.  Perhaps this matter is not at all related to plone/plone.base#18 .

**Comment by thet** [comment]
@yurj @Rudd-O yes, the tinymce-builded folder as well as the components folder are gone.
the URL‌ cannot work with current Plone 6 this way.

the only occurrence for ``tinymce-builded`` I found in my Plone 6 coredev buildout omelette tree is this:
```
thet@then:/home/_thet/data/dev/sprints/plone6ui/REPOS/buildout.coredev/parts/packages$ rg tinymce-builded
Products/CMFPlone/patterns/settings.py
160:            "loadingBaseUrl": "{}/++plone++static/components/tinymce-builded/"
```
But then, this is also the only occurrence of ``loadingBaseUrl``.

Any more information how to reproduce this problem or where the ``loadingBaseUrl`` or the ``tinymce-builded`` directory might be used?

**Comment by Rudd-O** [comment]
That needs fixin', I just don't know what to include there.

**Comment by jensens** [comment]
Closing as outdated. Mockup was completely rewritten for Plone 6 (jQuery/Backbone/RequireJS → ES6/Patternslib). TinyMCE visualblocks CSS URL from alpha2 era; path no longer exists.
If you think this issue is still relevant in the new implementation, please reopen it. Thank you for the original report!

**Comment by yurj** [comment]
I confirm this is fixed in current Classic UI demo site.
